src/core/services/value.js

```diff
@@ -19,7 +19,9 @@ export function getFactory(column) {
 			? row => column.value(row)
 			: column.path
 				? compileGet(column.path)
-				: row => row[column.key];
+				: row => row.rows
```

Comment:
this is wrong, what if user will pass object with property `rows`? you will give wrong value in this case, also why you need to make it right here?